catalog/app/data/workflows.spec.js

```diff
@@ -0,0 +1,108 @@
+import parseWorkflows, {
+  emptyWorkflowsConfig,
+  workflowNotAvaliable,
+  workflowNotSelected,
+} from './workflows'
+
+describe('Workflows config', () => {
```

Comment:
cascade of `describe`s and `it`s should form a proper sentence, e.g.
```javascript
describe('workflows.parse', () => {
  describe('when given an empty config file', () => {
    beforeEach(() => {
      // set things up
    })
    it('should return empty config', () => {
      // assertions
    })
    // it('should do something or be something', () => {})
  })
})
```
so it reads like "workflows.parse when given an empty config file should return empty config".
`describe` sets context (specify subject or condition), `it` defines an invariant